docs/DeveloperGuide.adoc

```diff
@@ -767,6 +828,43 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add person
+
+*MSS*
+
+1. User requests to add person
+2. AddressBook adds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The input parameters are invalid.
++
+[none]
+** 1a1. AddressBook shows an error message.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: List persons
+
+*MSS*
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
++
```

Comment:
Overall, the user case interactions are correct. However, more not similar/simple use cases are needed here.